routes/alumni.js

```diff
@@ -48,8 +48,8 @@ router.post('/', function(req, res, next) {
 
   function findEmptyField() {
     Object.keys(req.body.newAlumni).forEach((value) => {
-      console.log(value);
-      if (newAlumni[value] === "") {
+      console.log(newAlumni[value].length,value);
+      if (newAlumni[value]|| newAlumni[value].length <= 0) {
```

Comment:
why did you get rid of the `=== "" check on newAlumni[value]`?

Reply:
I meant to get rid of the === "" check since an empty string is falsey, but i forgot to put the '!' to evaluate the condition as true.